.travis.yml

```diff
@@ -142,6 +133,53 @@ jobs:
         - target/cli/linux/conduit install --conduit-version=$CONDUIT_TAG |tee conduit.yml
         - kubectl -n conduit apply -f conduit.yml --prune --selector='conduit.io/control-plane-component'
 
+    # Run integration tests after container images have been published.
+    - stage: integration-test
+
+      language: go
+      go: "1.10.2"
+      go_import_path: github.com/runconduit/conduit
+
+      cache:
+        directories:
+          - vendor
+          - "$HOME/google-cloud-sdk/"
+          - "$HOME/.cache"
+
+      install:
+        - ./bin/dep ensure -vendor-only -v
+        - ./bin/dep status -v
+
+        - |
+          # Install gcloud and kubectl.
+          dir="${CLOUDSDK_INSTALL_DIR:-${HOME}}/google-cloud-sdk"
+          (. bin/_gcp.sh ; install_gcloud_kubectl "$dir")
+          . "$dir/path.bash.inc"
+        - |
+          # Configure gcloud with a service account.
+          openssl aes-256-cbc -K $encrypted_6af64675f81c_key -iv $encrypted_6af64675f81c_iv -in .gcp.json.enc -out .gcp.json -d
+          (. bin/_gcp.sh ; set_gcloud_config "$GCP_PROJECT" "$GCP_ZONE" "$GKE_CLUSTER")
+        - |
+          # Get a kubernetes context.
+          (. bin/_gcp.sh ; get_k8s_ctx "$GCP_PROJECT" "$GCP_ZONE" "$GKE_CLUSTER")
+        - gcloud version
+        - kubectl version --short
+
+      script:
+        - |
+          # Run tests and cleanup afterward.
+          exit_code=0
+          cleanup_exit_code=0
+          version=$(./bin/go-run cli version --client --short)
+          namespace=conduit-$(echo $version | tr -cd '[:alnum:]-')
+          ./bin/test-run $(pwd)/.gorun $namespace || exit_code=$?
+          ./bin/test-cleanup $namespace || cleanup_exit_code=$?
```

Comment:
Consider that, if test-run ever hangs for 10 minutes, the cleanup code will not run!  Consider moving the cleanup logic into an after_* or post_* stage (instead of script)